src/PdV.cpp

```diff
@@ -83,10 +83,6 @@ void PdV_kernel(
 
 			double volume_change_s = volume[idx] / (volume[idx] + total_flux);
 
-			double min_cell_volume =
```

Comment:
I'm worried these are actually needed. We can't delete the computation as it needs to match exactly all the versions.

Reply:
I'll keep this but I'm pretty sure it gets compiled away as `min_cell_volume` was never used after declaration.